src/hearings/services/hearings.service.ts

```diff
@@ -52,4 +52,8 @@ export class HearingsService {
   public updateHearingActuals(hearingId: string, hearingActualsModel: HearingActualsModel): Observable<HearingActualsModel> {
     return this.http.put<HearingActualsModel>(`api/hearings/hearingActuals/${hearingId}`, hearingActualsModel);
   }
+
+  public submitHearingActuals(hearingId: string): Observable<any> {
```

Comment:
The any needs to be replaced with the actual model.

Reply:
I have updated the code to use number instead of any, as the API returns only the status.

Reply:
I have updated the code to use number instead of any, as the API returns only the status.